malaria-app-android/src/main/java/com/peacecorps/malaria/activities/MedicineStore.java

```diff
@@ -89,6 +92,29 @@ public void onClick(View view) {
                 final EditText medicineQuantityEt=(EditText)orderMedicineDialog.findViewById(R.id.order_quantity);
                 ((EditText)orderMedicineDialog.findViewById(R.id.order_quantity)).requestFocus();
                 //implement the email button
+
+                medicineQuantityEt.setOnKeyListener(new View.OnKeyListener() {
+
+                    @Override
+                    public boolean onKey(View v, int keyCode, KeyEvent event) {
+                        try {
+                            if (!medicineQuantityEt.getText().toString().equals("") &&
+                                    Integer.parseInt(medicineQuantityEt.getText().toString().trim()) > 100) {
+                                medicineQuantityEt.setError(String.valueOf(R.string.limit_exceeded));
```

Comment:
@HimanshuS01 this method of referencing the string value is showing the value stored in R file,instead use getResources().getString(_id_)